packages/mui-base/src/ToggleButton/index.ts

```diff
@@ -0,0 +1 @@
+export { Root as ToggleButton } from './index.parts';
```

Comment:
`export * as ToggleButton from './index.parts'`
TS has a meltdown when I do this, why?